target/linux/ramips/dts/mt7628an_tplink_tl-mr6400-v4.dts

```diff
@@ -89,8 +89,10 @@
 };
 
 &wmac {
-	nvmem-cells = <&macaddr_factory_1f100>;
-	nvmem-cell-names = "mac-address";
+	status = "okay";
+
+	nvmem-cells = <&eeprom_factory_20000>, <&macaddr_factory_1f100>;
```

Comment:
use mac-base syntax:
```
nvmem-cells = <&eeprom_factory_20000>, <&macaddr_factory_1f100 0>;
```
```
macaddr_factory_1f100: macaddr@1f100 {
		compatible = "mac-base";
		reg = <0x1f100 0x6>;
		#nvmem-cell-cells = <1>;
);
```

Reply:
Is this needed when we do not use any offset? 

We still have many places where we do not set `mac-base`.

Reply:
In general, dts files are a mess and various solutions are used (e.g. still use label instead of color/function for led). I believe that they should be unified and this can always be done along with other changes.